src/statsmodels/formula.jl

```diff
@@ -444,59 +455,79 @@ creating the model matrix.
 @compat function (::Type{ModelMatrix{T}}){T<:AbstractFloatMatrix}(mf::ModelFrame)
     dfrm = mf.df
     terms = droprandomeffects(dropresponse!(mf.terms))
+    factors = terms.factors
+
+    ## Step 1:
+    ## compute which columns correspond to which terms in order to pre-allocate
+    ## model matrix and set up views later
+    term_col_inds = []
+    num_cols = convert(Int, terms.intercept)
+
+    for (i_term, term) in enumerate(terms.terms)
+        f = Compat.view(factors, :, i_term)
```

Comment:
Actually, didn't we bump the required Julia to 0.5, making these unnecessary? (I discovered in trying to test this under 0.4.6...)